pkg/operations/normal.go

```diff
@@ -1086,7 +1185,18 @@ func readRandomlyInParallel(np *NormalProg) error {
 
 			var doc2 datagen.Doc
 			start := time.Now()
-			_, err := coll.ReadDocument(ctx, doc.Key, &doc2)
+
+			var err error
+			if np.UseAql {
+				query := "RETURN DOCUMENT(@myDocumentID)"
+				bindVars := map[string]interface{}{
+					"myDocumentID": coll.Name() + "/" + doc.Key,
+				}
+				_, err = db.Query(ctx, query, bindVars)
```

Comment:
We should ask somebody in the go team, if this is OK or if this prevents the cursor object from being closed. I do not know the consequences, but we should check with them.